PresentScreenings/Controllers/ScreeningDialogController.cs

```diff
@@ -329,6 +329,7 @@ public void UpdateAttendances()
             _presentor.UpdateAttendanceStatus(_screening);
             _presentor.ReloadScreeningsView();
             UpdateScreeningControls();
+            _labelTime.StringValue = _screening.ToLongTimeString();
             _screeningInfoControl.ReDraw();
```

Comment:
Please consider to update the name of this method as to fit the changed functionality.

Reply:
The name `UpdateAttendances()` was and is adequate for all calls within `ScreeningDialogController`.
I took the part out that updates the screening time, and put it into a new method `UpdateMovedScreeningInfo()` that also calls `UpdateAttendances()`.